ingestion-beam/src/main/java/com/mozilla/telemetry/decoder/ParseUri.java

```diff
@@ -288,7 +288,8 @@ private static class InvalidIntegerException extends InvalidUriException {
         putBoolPerCode(ImmutableMap.of("set_default", 2)), //
         putString("download_ip"), putString("attribution"),
         putIntegerAsString("profile_cleanup_prompt"), putBool("profile_cleanup_requested"),
-        putString("distribution_id"), putString("distribution_version"));
+        putString("distribution_id"), putString("distribution_version"),
+        putInteger("windows_ubr"), putString("stub_build_id"));
```

Comment:
I think this should fix the CI error
```suggestion
        putString("distribution_id"), putString("distribution_version"), putInteger("windows_ubr"), 
        putString("stub_build_id"));
```